hw/ip/uart/data/uart.hjson

```diff
@@ -179,13 +179,13 @@
       fields: [
         { bits: "0",
           swaccess: "wo",
-          hwaccess: "hro",
+          hwaccess: "hrw",
```

Comment:
I think you can now remove hwqe: "true" (at least I can't see fifo_ctrl using qe anywhere, so I think it was just there for these two bits)